ostree-ng.sh

```diff
@@ -35,12 +36,16 @@ CONTAINER_PUSHING_FEAT="false"
 TEMPDIR=$(mktemp -d)
 BLUEPRINT_FILE=${TEMPDIR}/blueprint.toml
 QUAY_CONFIG=${TEMPDIR}/quay_config.toml
+HTTPD_PATH="/var/www/html"
+KS_FILE=${HTTPD_PATH}/ks.cfg
 COMPOSE_START=${TEMPDIR}/compose-start-${IMAGE_KEY}.json
 COMPOSE_INFO=${TEMPDIR}/compose-info-${IMAGE_KEY}.json
+GRUB_CFG=${HTTPD_PATH}/httpboot/EFI/BOOT/grub.cfg
 
 # SSH setup.
 SSH_OPTIONS=(-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=5)
 SSH_KEY=key/ostree_key
+EDGE_USER_PASSWORD=foobar
```

Comment:
Why do you need this password? Thanks.

Reply:
We need it for ansible_become_pass variable. If you are agree, I will delete EDGE_USER_PASSWORD and add the ansible_become_pass=foobar directly.

Reply:
Because you set no sudo password needed in kickstart file, so do not need ansible become password.